src/configs/proposalTitles.json

```diff
@@ -1,66 +1 @@
-{
-  "0x08382f2197b647b338a15bcb33287dc3edc278f8c089a12952b87ee3fbadb42a": "Update swapr.eth resolver to point to beta 8.1 release of Swapr",
-  "0x57985dd9a8a45009b86a82a20d3669e570106014aa4b1b46bbc96875977a5c28": "REP Transfer - fluiddrop - MINT proposal",
-  "0x83c84a67ac418e87269150ca8c947e9f77f1d474124c4de59dd53dc251617059": "REP Transfer - fluiddrop - BURN proposal",

```

Comment:
@rossneilson not sure why this config files went through. Should remove them? or this is generate again later?

Reply:
Its generated when we build the cache but best option here is keeping it as it was before

Reply:
`git add -p` solves this issue.

Reply:
`git add -p` prevents similar issues occurring